src/System.ServiceModel.NetTcp/src/System.ServiceModel.NetTcp.Facade.csproj

```diff
@@ -32,11 +37,6 @@
     <Reference Include="System.IdentityModel" />
     <Reference Include="System.ServiceModel" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == 'net461'">
-    <PackageReference Include="System.ServiceModel.Primitives" Version="4.5.3" />
```

Comment:
Why the package reference is removed? Same comment applies to all files. From the previous released facade package, it had package dependency on System.ServiceModel.Primitives.

Reply:
The last released packages do have that dependency. If we keep them in these new packages it should be to the new 4.6.0 packages.
I will re-build these packages and re-run the manual test project and use a type in S.SM.Http that has a dependency to S.SM.Primitives.

Reply:
@ericstj If it is correct that .NETFramework, Version=v4.6.1 have a dependency to S.SM.Primitives and version 4.6.0 then I'm not sure how we did that in the buildtools world. Since it isn't built yet and therefore not available on any feed.

Reply:
The reference project should have a P2P reference to it.  We do that because a .NETStandardLibrary will need it, even if the facade doesn't actually refrence it (since it has no typedefs) we preserve the reference.  Outside the pkproj world you need to manually specify your project reference.

In general, be careful about conditioning refrences in the framework specific builds that are in the netstandard build: doing so could break the libraries who only build for netstandard but are consumed in netframework.